.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "porepy-stable",
+  "build": {
+    "dockerfile": "../dockerfiles/stable/Dockerfile"
+  },
+  "remoteUser": "root",
```

Comment:
The following settings need to be adapted if they change in the porepy image:

- `remoteUser` (I think you discussed adding some default user besides root)
- `workspaceFolder` (in case you change the directory of the porepy code)
- `defaultInterpreterPath` and `kernels.ExcludePythonEnvironments` whenever that changes. But please consider keeping the number of Python installation in the image at 1, I had a hard time to make this user-friendly.